scripts/populate_lambda_ssm_parameters.py

```diff
@@ -52,6 +52,11 @@ def set_deployed_lambda_environment(name, env: dict):
         }
     )
 
+def get_cache_criteria():
```

Comment:
do you have to open the file every time you need to check the criteria?

Reply:
negative, its only used to populate the Lambda ENV, which is then the lambda utilizes.